blocks/rich-text/style.scss

```diff
@@ -78,6 +78,19 @@
 	&.mce-content-body {
 		line-height: $editor-line-height;
 	}
+
+	div.mce-resizehandle {
```

Comment:
> Would be cool if it looks just like the image block resining handles?

It would be cool. It's not a huuuge necessity, but it would be a nice enhancement, in this or a separate PR. 

I haven't tested recently, but I could swear that I saw a commit fly by that mentioned this?

Reply:
Yes, I thought I had them looking identical in commit https://github.com/WordPress/gutenberg/pull/5794/commits/2bd82b75df1331a70327cd726eea6fead8594e36 @jasmussen, but I may have missed something. What was the difference you noticed @iseulde?

Reply:
Having them look different might actually be beneficial in communicating it's not a block. Maybe not entirely different, but white instead of blue.

Reply:
That makes sense. I tried out the white background in https://github.com/WordPress/gutenberg/pull/5794/commits/7fbaea2ebecc34354f75d05d5ba9cba1ccf218d0. Let me know what you think.

<img width="731" alt="screen shot 2018-05-03 at 8 24 07 pm" src="https://user-images.githubusercontent.com/10903492/39609055-f85db714-4f13-11e8-96e9-cd1ea7bb207a.png">

Reply:
Looks good! If needed we can adjust later.